src/pages/health-declaration/cat-adi-part2/health-declaration.cat-adi-part2.page.ts

```diff
@@ -53,8 +53,8 @@ interface HealthDeclarationPageState {
   candidateName$: Observable<string>;
   candidateUntitledName$: Observable<string>;
   candidateDriverNumber$: Observable<string>;
-  passCertificateNumber$: Observable<string>;
-  licenseProvided$: Observable<boolean>;
+  // passCertificateNumber$: Observable<string>;
+  // licenseProvided$: Observable<boolean>;
```

Comment:
same here, should this be removed?

Reply:
will be removed after i rebase and your changes are in!